big_tests/tests/gdpr_SUITE.erl

```diff
@@ -238,6 +244,12 @@ vcard_started() ->
 offline_started() ->
     dynamic_modules:ensure_modules(domain(), offline_required_modules()).
 
+private_required_modules() ->
+    [{mod_private, [{backend, pick_enabled_backend()}]}].
+
+private_started() ->
```

Comment:
Can it be included directly in a place it is called, because the method is not used any more?